core/serv-api/web/src/middleware/auth/mod.rs

```diff
@@ -65,9 +66,24 @@ where
     }
 
     fn call(&self, req: ServiceRequest) -> Self::Future {
+        #[derive(Deserialize)]
+        #[serde(rename_all = "camelCase")]
+        struct QueryAuth {
+            auth_token: String,
+        }
+
         let header = parse_auth::<Bearer, _>(&req)
             .ok()
-            .map(|b| b.token().to_string());
+            .map(|b| b.token().to_string())
+            .or_else(|| {
+                if Some("websocket".as_bytes()) == req.headers().get("upgrade").map(AsRef::as_ref) {
+                    web::Query::<QueryAuth>::from_query(req.query_string())
```

Comment:
It is not a huge concern, but query param `authToken` might be printed out in logs.
It is unusual to see `Authorization` header in logs.